index.html

```diff
@@ -14,14 +14,20 @@
             <div>   
             <img class="menu-icon" src="/images/Icon - Menu.svg" alt="hamburger list">
             </div>
+            <ul class="desktop-nav">
+                <li><a class="navbar-tools" href="#">Portfolio</a></li>
+                <li><a class="navbar-tools" href="#">About</a></li>
+                <li><a class="navbar-tools" href="#">Contact</a></li>
+            </ul>
         </nav>
     </header>
 
-        <section class="headline">  
-            <h1 id="h1">I'm Mariane Glad to see you!</h1>
+        <section class="headline">
+            <div class="headline-container">  
+            <h1 id="h1">I'm Mariane<br> Glad to see you!</h1>
             <p class="paragraph">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
-            <a  class="connect" href="">LET'S CONNECT</a>
-        <ul class="social-icons">
+            <h1><a  class="connect" href="">LET'S CONNECT</a></h1>
+            <ul class="social-icons">
             <li><a href="https://twitter.com/home" target="_blank" aria-label="twitter icon">
                 <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
                     <path d="M19.7587 0.394423C19.4253 -0.00424693 18.8587 -0.103914 18.4253 0.195088L16.492 1.52399C15.592 0.560536 14.3253 0.0289756 12.9587 0.0289756C10.2587 0.0289756 8.02532 2.22166 8.02532 4.94591C8.02532 5.34458 8.05865 5.71002 8.15865 6.07547C5.82532 5.57713 4.42532 3.88279 3.89198 0.826316C3.82532 0.427646 3.52532 0.128643 3.12532 0.0289756C2.75865 -0.070692 2.35865 0.0954206 2.12532 0.394423C2.05865 0.460868 -3.44135 8.10205 4.52532 14.5804C3.75865 16.2083 1.45865 17.6037 0.558651 18.0688C0.125318 18.2681 -0.0746825 18.7332 0.0253175 19.1984C0.125318 19.6635 0.525318 19.9625 0.991984 19.9625H2.72532C12.8587 19.9625 17.4587 14.9459 17.5587 3.71667L19.6587 1.72332C20.0587 1.3911 20.092 0.793093 19.7587 0.394423Z" fill="#505F79"/>
```

Comment:
- [ ] Please make sure for our headline section, The text and social media icon links are in the centre of the page but aligned to the left, currently they are aligned perfectly to the left but the content is not centered and is aligned to the bottom, this could be because I saw you are using position absolute to align the content. please try to fix his

![image](https://user-images.githubusercontent.com/97521557/208231393-5cb85cd9-2f20-4ba6-a91d-a73f3a815957.png)

As one of the goal of this project is to parse a figma design successfully, let's make our portfolio as close to the template as possible, see in the figma design how the text nd icons are centered on the page

![image](https://user-images.githubusercontent.com/97521557/208231515-398c25e4-3d6d-41ed-961f-554fcf2f4fb0.png)

Reply:
Thank you for the feedback, align as recommended.